api/types/types.go

```diff
@@ -234,6 +235,10 @@ const (
 	ServiceSyncAddress = "mc/$skupper-service-sync"
 )
 
+const (
+	SkupperServiceCertPrefix string = "skupper-"
```

Comment:
One minor question here.
Is it safer/better to be more specific on the prefix, like: `skupper-tls-`?

The code is already validating if there is a conflict, but using a more specific
prefix, it is easier to list "all" secrets related to tls services.

Reply:
Sounds good, could be useful when executing skupper delete command

Reply:
Done.